test/Confluent.Kafka.IntegrationTests/TemporaryTopic.cs

```diff
@@ -23,19 +23,24 @@ namespace Confluent.Kafka
 {
     public class TemporaryTopic : IDisposable
     {
-        IAdminClient adminClient;
+        private string bootstrapServers;
```

Comment:
changes to this class are probably not necessary. I was thinking it would be better (more focussed test) if the adminclient was'd open over the duration of the lifetime of the temporary topic. but the improvement is marginal really. anyway, since i made the change, I left it in.